pytom/alignment/GLocalSampling.py

```diff
@@ -805,13 +807,12 @@ def averageParallel(particleList,averageName, showProgressBar=False, verbose=Fal
     splitLists = splitParticleList(particleList, setParticleNodesRatio=setParticleNodesRatio)
     splitFactor = len(splitLists)
     assert splitFactor > 0, "splitFactor == 0, issue with parallelization"
-    print(f'Device = {device}')
     if 'gpu' in device:
         from pytom.bin.average import averageGPU as average
         from pytom.agnostic.structures import Reference
         from pytom.agnostic.io import read, write
 
-        print('Averaging volumes on gpu.')
+        print(f'averaging particles on {device} for {averageName}.')
```

Comment:
nitpick, start sentence with a capital
```suggestion
        print(f'Averaging particles on {device} for {averageName}.')
```